packages/teleport/src/DesktopSession/useClipboard.ts

```diff
@@ -0,0 +1,110 @@
+import { useState, useEffect } from 'react';
+
+export function useClipboardReadWrite(
+  shouldSetOrPrompt: boolean
```

Comment:
it wasn't clear what `shouldSetOrPrompt` meant
```suggestion
  canUseClipboardReadWrite: boolean
```